doc/src/arch/reference.rst

```diff
@@ -1416,7 +1416,7 @@ The following tags are common to all <pb_type> tags:
     Multple clock ports are described using multiple ``<clock>`` tags.
     *See above descriptions on inputs*
 
-.. arch:tag:: <mode name="string">
+.. arch:tag:: <mode name="string" packable="bool">
```

Comment:
Since we expect typical usage to not specify packable, perhaps we should give it a more exceptional name.

Perhaps something like `disable_pack` or `disable_packing`. Not sure

Reply:
This is a good point. I will change the name.